src/plugins/intel_cpu/src/config.cpp

```diff
@@ -253,6 +253,9 @@ void Config::readDebugCapsProperties() {
     if (envVarValue = readEnv("OV_CPU_BLOB_DUMP_NODE_NAME"))
         blobDumpFilters[BY_NAME] = envVarValue;
 
+    if (envVarValue = readEnv("OV_CPU_PRINT_CONSTANT_NODES"))
+        shouldPrintConstNodes = envVarValue;
```

Comment:
I don't we need this config param right now. It is required for exec_graph, because sometimes information about constants might be valuable there. For perf counters it is not. So I propose to delete the key. 
@EgorDuplensky any comments from your side?

Reply:
Can we fix the test then?